lib/docker_manager/upgrader.rb

```diff
@@ -257,9 +257,9 @@ def status(val)
     publish("status", val)
   end
 
-  def log_version_upgrade
+  def log_version_update
```

Comment:
We renamed the method, but I don’t see another instance of the method in the list of changes. Do we have more places to update?

Reply:
I didn't intent to update that method name too. I fixed it now. I doubled checked all other method names again.